pyctcdecode/tests/test_decoder.py

```diff
@@ -600,3 +600,29 @@ def test_serialization(self):
 
         # this decoder has a language model so we expect one more key
         self.assertEqual(old_num_models + 1, self._count_num_language_models())
+
+    def test_load_from_hub_offline(self):
+        # test local loading
+        self.clear_dir()
```

Comment:
why is this `clear_dir()` needed. AFAIK this should be done before this test method executes